apps/openssl.c

```diff
@@ -243,7 +222,7 @@ int main(int argc, char *argv[])
     arg.size = 0;
 
     /* Set up some of the environment. */
-    default_config_file = make_config_name();
+    default_config_file = CONF_get1_default_config_file();
```

Comment:
this change is problematic, while make_config_name uses app_malloc
was at least in the past guaranteed to return non-zero.
I see OPENSSL_strdup was added and breaks that contract.
CONF_get1_default_config_file does not kill the process when out of memory happens.
so just add a fatal error handling here?

Reply:
Good point, will add that.

Reply:
Fixed